Learning_Guide/Goals/Improvement.md

```diff
@@ -0,0 +1,14 @@
+# Goal Improvewment
+
+
+## Submitting a PR against a Goal
+
+0. Go [here](https://github.com/GuildCrafts/web-development-js/tree/master/_goals)
+0. Find your goal and click on it
+0. Click the Pencil
+0. Edit the Markdown file. _either make changes or ask for the changes you want_
+0. **IMPORTANT** make sure you select `Create a new branch for this commit and start a pull request`
+0. Click `Propose file change`
+
+
+_Note: It can take as much as a few weeks for your change requests to be addressed. Please be patient_
```

Comment:
Really? That's unfortunate if that's the impression. I've tried to address PRs within 24 hours, don't think I've missed any so far.

Issues, on the other hand, are stacking up for a long time into the future.